<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Ankitkr" failures="1" tests="6" name="Test" time="217.82" errors="0" timestamp="2025-05-31T09:23:21 IST">
  <testcase classname="TestCases.TC_01registeration" name="verify_account_registration" time="10.017"/>
  <testcase classname="TestCases.TC_01registeration" name="verify_account_registration" time="13.5"/>
  <testcase classname="TestCases.TC_06EndtoEnd" name="verify_EndtoENd" time="54.328">
    <failure type="java.lang.AssertionError" message="Expected test to pass, but it failed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Expected test to pass, but it failed. expected [true] but found [false]
at TestCases.TC_06EndtoEnd.verify_EndtoENd(TC_06EndtoEnd.java:124)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verify_EndtoENd -->
  <testcase classname="TestCases.TC_06EndtoEnd" name="verify_EndtoENd" time="18.706"/>
  <testcase classname="TestCases.TC_06EndtoEnd" name="verify_EndtoENd" time="74.331"/>
  <testcase classname="TestCases.TC_06EndtoEnd" name="verify_EndtoENd" time="14.402"/>
</testsuite> <!-- Test -->
